core/commands/generate_command.rb

```diff
@@ -67,4 +72,21 @@ def read_template
       Result.ok('Template read')
     end
   end
+
+  # Read the template file and return all shared disks to create
+  def fetch_template_disks
```

Comment:
This code is for Vagrant only. It should be moved to the VargantConfigurationGenerator class.